src/main/java/com/google/javascript/clutz/ImportRenameMapBuilder.java

```diff
@@ -7,7 +7,19 @@
 import java.util.HashMap;
 import java.util.Map;
 
+/**
+ * If Clutz is running in incremental mode, closure can't resolve imported symbols, so it gives them
+ * names in the current namespace (ie `module$contents$current$file`), instead of names that
+ * reference the original namespace (ie `module$exports$imported$file`). ImportRenameMapBuilder
+ * builds a map from the local name to the exported name so the original names can be substituted.
+ */
 public class ImportRenameMapBuilder {
+
+  /**
+   * Takes a collection of source files, parses them with the compiler, and walks the ast to find
+   * any `const variable = goog.require()` statements to build a map from the name closure generates
+   * in incremental mode to the exported name of the import.
+   */
```

Comment:
`ImportRenameMapBuilder.buildImportRenameMap` is a bit of a mouth full and won't help readers much. What about just calling this `.build`?

Reply:
Sounds good!